Framework/Foundation/include/Framework/Signpost.h

```diff
@@ -25,7 +25,7 @@
 #if defined(__APPLE__) && __has_include(<os/signpost.h>) && (__MAC_OS_X_VERSION_MAX_ALLOWED >= __MAC_10_15)
 #include <os/signpost.h>
 #include <os/log.h>
-#ifdef OS_LOG_TARGET_HAS_10_15_FEATURES
+#if defined(OS_LOG_TARGET_HAS_10_15_FEATURES) && defined(OS_LOG_CATEGORY_DYNAMIC_TRACING)
```

Comment:
Any idea why OS_LOG_CATEGORY_DYNAMIC_TRACING is not there? As far as I understand OS_LOG_TARGET_HAS_10_15_FEATURES implies OS_LOG_CATEGORY_DYNAMIC_TRACING.

Reply:
It does not. My compiler had the first one defined and not the second one. Maybe just checking for the second one would be much more direct in any case.